libheif/heif.h

```diff
@@ -497,6 +497,13 @@ struct heif_error heif_context_get_image_handle(struct heif_context* ctx,
 LIBHEIF_API
 void heif_context_debug_dump_boxes_to_file(struct heif_context* ctx, int fd);
 
+// Created by Devon Sookhoo
```

Comment:
Let's not put our names in the source code.  It should be able to stand long after we are done with it.